File: sentimental-mario-less/mario.py
import cs50

height = 0

# Loop to validate the value of Height
while height > 8 or height < 1:
    height = cs50.get_int("Height: ")

#Pyramid
for i in range(height):
    for j in range(height):
        if j < (height - i - 1):
            print(" ", end='')
        else:
            print("#", end='')
    print()

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height <= 0);

    // Print made

    // Left Side
    for (int row = 0; row < height; row++)
    {
        for (int collum = 0; collum < height; collum++)
        {
            if (collum < height - row - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        printf("  "); // Space between sides

    // Right side
        for (int collum = 0; collum < row + 1; collum++)
        {
            printf("#");
        }

        printf("\n");
    }
}